### ISSUE TITLE
Error when entering wrong password to login into CP

### ISSUE THREAD

**Issue by smcyr** [statamic/cms#952]
I have this error when I enter the wrong password into the CP:
```
ErrorException
file_put_contents(D:\projects\test\storage\framework/cache/data/email@email.com|192.168.1.XXX/e5/c5/e5c593373173624acd2c2af96207c7cd): failed to open stream: No such file or directory 

Illuminate\Foundation\Bootstrap\HandleExceptions::handleError :133
D:\projects\mutation\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:133
```

I'm on Windows 10, so it's maybe a path problem?

**Comment by jasonvarga** [comment]
What's the `XXX.XXX.X.XXX` bit? Are there any `::` in it?

**Comment by smcyr** [comment]
It's my local IP which is 192.168.1.XXX

**Comment by ddtigner** [comment]
I'm receiving the same error. After a bit of testing, it appears to be related to Windows. I did a fresh install on my Mac and on Windows. The Mac doesn't have the problem.

Windows 10, latest update, using Laragon.

You can type anything into the username with no password and it works as expected (the screen shakes). As soon as you provide a password that's incorrect, the error is displayed. If you use the correct username and password, the control panel loads correctly.

**Comment by amadeann** [comment]
I'm also getting the same error, and from what I remember the same thing happened to me with v2 as well. It's probably because Windows doesn't allow pipes in directory names.

`Statamic\Auth\ThrottlesLogins@incrementLoginAttempts` sets the cache key with `|` and then `Statamic\Extensions@path` takes that cache key, and tries to use it as a directory name.

The solution could be replacing `|` with `/` inside `Statamic\Extensions@path`, but I have no idea whether other parts of the app depend on `|` in the cache directories names.

**Comment by innocenzi** [comment]
I have this issue on Windows as well, when entering a wrong password. That pipe is indeed problematic.

**Comment by jasonvarga** [comment]
Sorry this issue has been so neglected. It's fixed now and will be available in 3.0.1.
